### DISCUSSION TITLE
Tiles

### DISCUSSION THREAD

**Discussion by thomasnik** [TerriaJS/terriajs#6818] [Q&A]
Hello all,

I would like to change the tiles of the app and use the photorealistic 3D tiles from google. 
I'm searching into the files for a long time and I tried to change the file CesiumedTilesMixin, private loadTileset function, this.url with my the url provided by photorealistic 3d maps of google. When i did it and i run it i got no error but nothing change.

Can anyone help me or advise me what I have to do to change the tiles??

Thank you in advance!

**Comment by techmavengeospatial** [comment]
There is no code to change
this is just an init JSON (Catalog) configuration.
https://docs.terria.io/guide/connecting-to-data/catalog-type-details/3d-tiles/

{
  "catalog": [
    {
      "type": "3d-tiles",
      "url": "https://tile.googleapis.com/v1/3dtiles/root.json?key=[yourkey]",
      "name": "google_3d_tiles",
      "opacity": 1,
      "description": "Google 3D Tiles",
      "hideSource": true
    }
  ],
  "homeCamera": {
    "north": 48.709088,
    "east": -64.055099,
    "south": 25.507742,
    "west": -129.445724
  },
  "initialCamera": {
    "north": 48.709088,
    "east": -64.055099,
    "south": 25.507742,
    "west": -129.445724
  }
}

If you want to view it on iOS and Android
check out https://earthexplorer.techmaven.net

**Reply by thomasnik** [reply]
Hello and thank you for you response, I'm trying to find this catalog file. When i go to wwroot and then init folder and then simple.json there is a catalog but it has not the above format. Where can i find this catalog.json file??
